packages/Button/Button.md

```diff
@@ -5,13 +5,17 @@ Provide a function as the `onClick` prop to perform an action when clicked. **Av
 ### Usage criteria
 
 - Use buttons to move through a transaction
-- Aim to use only one primary button per page
-- Keep the text short and able to fit on a single line (our recommendation is 2-24 characters)
+- Aim to use only one primary button variant per page
+- Keep the text short and precise. It should fit on a single line (our recommendation is 2-24 characters for French and English)
+- Make use of the A11yContent core component to provide more written context for assistive technology users for when they navigate a page using only button landmarks
 - Make sure the button text describes an action
 - Buttons should not be disabled. Use inline (on blur) error messaging to provide feedback when the form is invalid
 
-By default, Buttons will be displayed in the `primary` variant. Use primary buttons for the main action on a page or
-in a form.
+### OLD Button variants
+
+**NOTE:** New Button variants have been introduced. Future projects should adopt the new variants, as the older variants will have a deprecation plan. Avoid using both new and old variants together within the same experience.
```

Comment:
Can we change this to:

**Deprecated:** New Button variants have been introduced. Future projects should adopt the new variants, as the older variants will be deprecated. Avoid using both new and old variants together within the same experience.

Reply:
done :)